src/aead/aes_gcm.rs

```diff
@@ -52,20 +54,30 @@ fn aes_gcm_seal(ctx: &[u64; aead::KEY_CTX_BUF_ELEMS],
                 -> Result<(), error::Unspecified> {
     let ctx = polyfill::slice::u64_as_u8(ctx);
     bssl::map_result(unsafe {
-        GFp_aes_gcm_seal(ctx.as_ptr(), in_out.as_mut_ptr(), in_out.len(), tag,
-                         nonce, ad.as_ptr(), ad.len())
+        GFp_aes_gcm_seal(ctx.as_ptr(),
+                         in_out.as_mut_ptr(),
+                         in_out.len(),
+                         tag,
+                         nonce,
+                         ad.as_ptr(),
+                         ad.len())
```

Comment:
This is most of the reason why we don't use rustfmt already. We need a formatting tool that it can format function calls in the compact format, and that generally optimizes for readability first and minimum number of lines second, like the Google clang-format style.

Alternatively, we might be able to use the default rustfmt style after we've oxidized more of *ring* so that there aren't so many function calls that take a large number of arguments. But not yet.